qgreenland/config/helpers/layers/lonlat.py

```diff
@@ -7,6 +7,13 @@
 from qgreenland.models.config.layer import ConfigLayer, ConfigLayerInput
 
 
+lonlat_assets_sorted = sorted(
+    dataset.assets.values(),
+    key=lambda asset: float('.'.join(asset.id.split('_')[1:-1])),
```

Comment:
This is a little icky... one line is doing three major things: Getting out the numerical portion of the ID, and replacing the underscore with a dot, and casting the result to a float for sorting. A float is not a good representation of the underlying data, but for sorting it's fine and it's possibly faster to cast to a float and compare than to cast to `Decimal` and compare. No idea, really, just a guess..